flask/README.md

```diff
@@ -1,24 +1,124 @@
 
 ## Readme
 
-This project is developed in Python >= 3.6. For setting up the development environment for the first time, create a [Python virtual environment](https://docs.python.org/3.6/library/venv.html), e.g. by
+## Table of Contents
+1. [Contribution Guidelines](../CONTRIBUTING.md)
+2. [Development Set-up](#development-set-up)
+    1. [Install python](#install-python)
+    2. [Set-up pre-commit](#set-up-pre-commit)
+    3. [Linting and Formatting in VSCode](#linting-and-formatting-in-vscode)
+    4. [Working with MySQL](#working-with-mysql)
+    5. [Debugging](#debugging)
+3. [Testing][#testing]
```

Comment:
use ( ) instead of [ ] for correct markdown

Reply:
done